### ISSUE TITLE
No support for Ultimax cartridges

### ISSUE THREAD

**Issue by fade0ff** [GeorgRottensteiner/C64Studio#26]
Currently, it's not possible to directly create a CRT for ultimax mode. Using cart8crt almost works, but "EXROM" isn't set, i.e. at offset 0x18, 0x00 should be 0x01.
Also the loading start address is wrong. At offset 0x4c, 0x80 (->0x8000) should be 0xe0 (->0xe000).

It should suffice to add a "ultimaxcrt" keyword which uses the following lines at the according locations: 

          header.AppendU8( 1 );     // EXROM
          header.AppendU8( 0 );     // GAME
          ...
          chip.AppendU16NetworkOrder( 0xE000 ); // loading start address
          chip.AppendU16NetworkOrder( 0x2000 ); // rom size

**Comment by GeorgRottensteiner** [comment]
Added with commit b55ff0af, currently untested though. Support adds 4k, 8k and 16k Ultimax formats.

**Comment by fade0ff** [comment]
Thanks a lot for the quick change. There are no nightly build somewhere, or are there?

**Comment by fade0ff** [comment]
OK, Sorry, forget it, it was easier to build than I imagined.
The Ultimax support didn't work though at first try. I fixed a few issues in ASMFileParser.cs (renamed to txt to upload it here):
[ASMFileParser.txt](https://github.com/GeorgRottensteiner/C64Studio/files/3919428/ASMFileParser.txt)
With these fixes, at least ULTIMAX8CRT worked for me (didn't test the other settings).
Side note: it would be handy to create a BIN and CRT at the same time.
